components/CIResponsiveLine.tsx

```diff
@@ -0,0 +1,50 @@
+import { ResponsiveLine, Datum } from "@nivo/line";
+import { format, parse } from "date-fns";
+
+const CIResponsiveLine = ({ data }: Datum) => {
+  return (
+    <>
+      <div style={{ height: 400, width: "100%" }}>
+        <ResponsiveLine
+          data={data}
+          margin={{ top: 50, right: 150, bottom: 50, left: 60 }}
+          yScale={{
+            type: "linear",
+          }}
+          motionConfig="stiff"
+          curve="catmullRom"
+          enableSlices="x"
+          axisTop={null}
+          isInteractive={true}
+          axisRight={null}
+          axisLeft={{
+            tickValues: 5,
+            tickSize: 0,
+            tickPadding: 5,
+            tickRotation: 0,
+            legend: "Pull Requests Merged per Day",
+            legendOffset: -40,
+            legendPosition: "middle",
+          }}
+          axisBottom={{
+            tickSize: 0,
+            format: (value) => {
+              return format(parse(value, "dd/MM/yyyy", new Date()), "MMM d");
```

Comment:
```suggestion
              return format(parse(value, "MM/dd/yyyy", new Date()), "MMM d");
```
This fixes the build error.